lib/create-plugin.js

```diff
@@ -191,14 +191,18 @@ function createPlugin(instrumentationApi, config = {}) {
               resolverSegment.addAttribute(RETURN_TYPE_ATTR, info.returnType.toString())
               resolverSegment.addAttribute(PARENT_TYPE_ATTR, info.parentType.toString())
 
-              for (const [key, value] of Object.entries(args)) {
-                // Require adding to attribute 'include' configuration
-                // so as not to accidentally send senstive info to New Relic.
-                resolverSegment.attributes.addAttribute(
-                  DESTINATIONS.NONE,
-                  `${FIELD_ARGS_ATTR}.${key}`,
-                  value
-                )
+              // For ease of use, we'll add the arguments to both the
```

Comment:
this comment is confusing without the context of addressing this ticket.  
```suggestion
              // Like our http and framework instrumentation, we'll add the attributes on the operation segment.  Additionally we will add the attributes to resolver segments as they help inform performance impacts.
              ```

Or something to that effect

Reply:
Thank you, will rewrite.